tensorflow/core/common_runtime/next_pluggable_device/next_pluggable_device_factory.h

```diff
@@ -33,18 +34,22 @@ class NextPluggableDeviceFactory : public DeviceFactory {
       const std::string& compilation_device_name)
       : api_(TfnpdApi()),
         device_type_(device_type),
-        compilation_device_name_(compilation_device_name) {}
+        compilation_device_name_(compilation_device_name),
```

Comment:
I think compilation_device_name_ has the same meaning as jit_device_type_ so we do not need to add a new field jit_device_type_. We can wait to make this change until there is a need to separate compilation_device_name_ and jit_device_type_.

Reply:
done